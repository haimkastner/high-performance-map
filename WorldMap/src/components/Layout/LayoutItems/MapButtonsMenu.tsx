import React from "react";
import { MoveButton } from "../../Content/MapButtons/MoveButton";
import { MeasurementButton } from "../../Content/MapButtons/MeasurementButton";
import { EditButton } from "../../Content/MapButtons/EditButton";
import { ShipButton } from "../../Content/MapButtons/ShipButton";
import { ZoomInButton } from "../../Content/MapButtons/ZoomInButton";
import { ZoomOutButton } from "../../Content/MapButtons/ZoomOutButton";
import { Box, Grid, Theme, makeStyles, createStyles } from "@material-ui/core";

const useStyles = makeStyles((theme: Theme) =>
  createStyles({
    root: {
      width: "100%",
      height: "100%"
    }
  })
);

export function MapButtonsMenu() {
  const classes = useStyles();

  return (
    <Box>
      I am the MapButtonsMenu
    </Box>
  );
}
