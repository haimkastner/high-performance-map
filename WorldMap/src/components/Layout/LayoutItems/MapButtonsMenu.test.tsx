import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { MapButtonsMenu} from './MapButtonsMenu';

describe('MapButtonsMenu Component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<MapButtonsMenu />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('matches the snapshot', () => {
    const tree = renderer.create(<MapButtonsMenu />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  // tslint:disable-next-line: no-empty
  describe('MapButtonsMenu Component Logic', () => {});
});
