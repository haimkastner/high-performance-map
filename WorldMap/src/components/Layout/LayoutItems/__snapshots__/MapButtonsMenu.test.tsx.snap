// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapButtonsMenu Component matches the snapshot 1`] = `
<div
  className="MuiBox-root MuiBox-root-4"
>
  I am the MapButtonsMenu
</div>
`;
